src/java.base/share/classes/java/io/Reader.java

```diff
@@ -140,6 +140,125 @@ public void close() {
         };
     }
 
+    /**
+     * Returns a {@code Reader} that reads characters from a
+     * {@code CharSequence}, starting at the first character in the sequence.
+     *
+     * <p> The resulting reader is not safe for use by multiple
+     * concurrent threads. If the reader is to be used by more than one
+     * thread it should be controlled by appropriate synchronization.
+     *
+     * <p> The returned reader is initially open. The reader is closed by
+     * calling the {@code close()} method. Subsequent calls to {@code close()}
+     * have no effect.
+     *
+     * <p> After the reader has been closed, the {@code read()},
```

Comment:
The API docs in 5cbc0450 looks okay. One minor comment is that "After the reader has been closed ..." doesn't need to be a new paragraph as it's part of the lifecycle topic in the previous paragraph.

Reply:
IIUC I shall just remove the `<p>` but keep the text `After the reader...`?

Reply:
Yep. Better keep everything about closing in the same paragraph.